app/shared/src/forms/datePicker/DatePicker.android.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+import * as React from 'react';
+import { DatePickerAndroid } from 'react-native';
+
+import DatePickerButton from './DatePickerButton';
+
+type Props = {
+  date: Date,
+  format?: string,
+  onDateChange: (date: Date) => void,
+  minDate?: Date,
+  maxDate?: Date,
+};
+
+export default class DatePicker extends React.Component<Props> {
+  openDatePicker = () => {
+    // Touchable needs a method not returning a promise
```

Comment:
I do not understand this comment. Maybe please try to clarify it differently.

Reply:
I'll sign up for the english course as well 😞 I will improve the comment

Reply:
@tbergq No, it's fine. I am just not sure about the reason and why you actually pointed this out... :) Just make it more clear for your future @tbergq so he will understand immediately what is happening here... :)